code_search/docker/ks/submit_code_embeddings_job.sh

```diff
@@ -21,37 +21,38 @@ workerMachineType=n1-highcpu-32
 
 usage() {
 	echo "Usage: submit_code_embeddings_job.sh
-	--workflowId=<workflow id invoking the container>
-	--modelDir=<directory contains the model>
+	--cluster=<cluster to deploy job to>
 	--dataDir=<data dir>
-	--functionEmbeddingsDir=<output function embedding dir>
-	--tokenPairsBQTable=<input token pairs BQ table>
 	--functionEmbeddingsBQTable=<output function embedding BQ table>
+	--functionEmbeddingsDir=<output function embedding dir>
+	--modelDir=<directory contains the model>
+	--namespace=<kubernetes namespace>
 	--numWorkers=<num of workers>
 	--project=<project>
+	--timeout=<timeout>
```

Comment:
Why do we need a timeout?

Reply:
this controls how long the argo step waits on the dataflow job.

Reply:
Why isn't the timeout in the Argo Step?

Reply:
It's not yet supported by DSL. Add issue to track the item
https://github.com/kubeflow/pipelines/issues/462